tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/HotChocolateTests.cs

```diff
@@ -227,6 +238,123 @@ private void SubmitRequest(int aspNetCorePort, RequestInfo requestInfo, bool pri
             }
         }
 
+        private async Task SubmitWebsocketRequest(int aspNetCorePort, RequestInfo requestInfo)
```

Comment:
Would it be possible to factorize this code with the GraphQL tests? It's maybe not possible but there's a lot of common logic it seems so if it makes sense it could be worth doing it